src/main/java/seedu/address/logic/commands/DeleteCommand.java

```diff
@@ -3,51 +3,61 @@
 import static java.util.Objects.requireNonNull;
 
 import java.util.List;
+import java.util.Set;
 
 import seedu.address.commons.core.Messages;
 import seedu.address.commons.core.index.Index;
 import seedu.address.logic.commands.exceptions.CommandException;
 import seedu.address.model.Model;
 import seedu.address.model.person.Person;
+import seedu.address.model.person.UniquePersonList;
 
 /**
- * Deletes a person identified using it's displayed index from the address book.
+ * Deletes all internships identified using their displayed index from the PleaseHireUs
  */
 public class DeleteCommand extends Command {
 
     public static final String COMMAND_WORD = "delete";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD
-            + ": Deletes the person identified by the index number used in the displayed person list.\n"
-            + "Parameters: INDEX (must be a positive integer)\n"
-            + "Example: " + COMMAND_WORD + " 1";
+            + ": Deletes the internships identified by the index numbers used in the displayed internship list.\n"
+            + "Parameters: INDEX... (all indexes must be a positive integer)\n"
+            + "Example: " + COMMAND_WORD + " 1 3";
 
-    public static final String MESSAGE_DELETE_PERSON_SUCCESS = "Deleted Person: %1$s";
+    public static final String MESSAGE_DELETE_PERSON_SUCCESS = "Deleted Internship(s):\n%s";
 
-    private final Index targetIndex;
+    private final Set<Index> targetIndexes;
 
-    public DeleteCommand(Index targetIndex) {
-        this.targetIndex = targetIndex;
+    public DeleteCommand(Set<Index> targetIndexes) {
+        this.targetIndexes = targetIndexes;
     }
 
     @Override
     public CommandResult execute(Model model) throws CommandException {
         requireNonNull(model);
         List<Person> lastShownList = model.getFilteredPersonList();
+        UniquePersonList personsToRemove = new UniquePersonList();
 
-        if (targetIndex.getZeroBased() >= lastShownList.size()) {
-            throw new CommandException(Messages.MESSAGE_INVALID_PERSON_DISPLAYED_INDEX);
+        for (Index selectedIndex : targetIndexes) {
+            if (selectedIndex.getZeroBased() >= lastShownList.size()) {
+                throw new CommandException(Messages.MESSAGE_INVALID_PERSON_DISPLAYED_INDEX);
+            }
+
+            Person personToDelete = lastShownList.get(selectedIndex.getZeroBased());
+            personsToRemove.add(personToDelete);
+        }
```

Comment:
If you decide to create a new class called Indexes or Indexlist, I think this whole chunk can be extracted out